efel/cppcore/LibV1.cpp

```diff
@@ -181,7 +181,16 @@ int LibV1::ISI_values(mapStr2intVec& IntFeatureData,
     GErrorStr += "\n Three spikes required for calculation of ISI_values.\n";
     return -1;
   }
-  for (size_t i = 2; i < pvTime.size(); i++)
+
+  int IgnoreFirstSpike;
+  vector<int> retIgnore;
+  retVal = getIntParam(IntFeatureData, "ignore_first_spike", retIgnore);
+  if ((retVal < 0) || (retIgnore[0] > 1) || (retIgnore[0] < 0))
```

Comment:
I just wonder what 'retIgnore[0] < 0' means. Normally retignore will contain the python value (0 or 1?). 
Btw, are we sure retIgnore[0] will always exist, since the vector is uninitialized?

Can you also add a test for this setting.